.github/workflows/publish-code.yml

```diff
@@ -72,3 +72,26 @@ jobs:
       - name: Publish Nuget packages to Nuget registry
         working-directory: src
         run: dotnet nuget push "out/*" -k ${{secrets.NUGET_AUTH_TOKEN}}
+      - name: Zip documentation
+        working-directory: docs2/public
+        run: zip -r ../../src/out/Documentation.zip "./*"
+      - name: Publish release artifacts
+        uses: actions/github-script@v2
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            console.log('environment', process.versions);
+            const fs = require('fs').promises;
+            const { repo: { owner, repo }, sha } = context;
+
+            for (let file of await fs.readdir('src/out')) {
+              console.log('uploading', file);
+
+              await github.repos.uploadReleaseAsset({
+                owner,
+                repo,
+                release_id: ${{ github.event.release.id }},
```

Comment:
Is `github.event.release.id` correct?

Reply:
It would appear so, like how it is used here: https://github.com/marketplace/actions/release-feedback